src/parsers/xyz.js

```diff
@@ -140,10 +141,25 @@ function fromMaterial(materialOrConfig, fractional = false) {
     return fromBasis(basis, "%11.6f");
 }
 
+export function atomsCount(xyzFile) {
+    const xyzArray = xyzFile.split(/\r?\n/);
+    return Integer.parseInt(xyzArray[0]);
```

Comment:
1. Why do we even need this function, if all this does is just split on newlines? This is not specific to XYZ format, is it?
2. No docstring, explanation of the expected behavior

Reply:
The first line of an xyz file contains the number of atoms. So we split the string by new lines and then return the first new line to get the number of atoms in the file.